src/value.rs

```diff
@@ -947,8 +947,8 @@ impl Padding for PaddedEncoding {
 }
 
 impl Value {
-    fn from_bits<I: Iterator<Item = bool>, P: Padding>(
-        bits: &mut I,
+    fn from_bits<I: Iterator<Item = u8>, P: Padding>(
+        bits: &mut BitIter<I>,
```

Comment:
386622530d3f956ca19d74feed7759b3c69fbf3a: This breaks downstream code in the web IDE Bit Machine. However, since `Value` now contains `Arc<[u8]>` it makes sense to use an iterator over something like `[u8]`. In the web IDE, I can collect the bits in `Vec<u8>` and create a `BitIter` from that.

Reply:
Yeah, can you just collect into a `Vec<u8>` for now and raise an issue about it? After doing this I really want to revisit the `BitIter` API here. It's awkward in several ways.